katharsis-core/src/main/java/io/katharsis/queryspec/DefaultQuerySpecDeserializer.java

```diff
@@ -1,20 +1,14 @@
 package io.katharsis.queryspec;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
While I don't have an opinion on the tabs vs spaces spectacle, I do belive that * imports are more readable save dev time (hence more useful) when you have > X imports from the same package. The default intellij setting is >= 5 imports.